src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -132,7 +139,8 @@ Status CreateRequestQueue::ProcessRequests() {
       } else {
         if (plasma_unlimited_) {
           // Trigger the fallback allocator.
-          status = ProcessRequest(*request_it, /*fallback_allocator=*/true);
+          status = ProcessRequest(/*fallback_allocator=*/true, *request_it,
+                                  &spilling_required);
```

Comment:
```suggestion
                                  /*spilling_required=*/nullptr);
```
To indicate that this out-parameter will not be used.